### ISSUE TITLE
Switch to the new attrs(props => ({})) syntax instead for easier and more powerful composition.

### ISSUE THREAD

**Issue by sanishkr** [sag1v/react-elastic-carousel#25]
Warning:

```
Functions as object-form attrs({}) keys are now deprecated and will be removed 
in a future version of styled-components. Switch to the new attrs(props => ({})) 
syntax instead for easier and more powerful composition. 
The attrs key in question is "style" on component "styled.div". 
 Error
    at ...........
```

Resolution required in 2 files:

1. In Poster.js

```
import styled from "styled-components";
--
2 |  
3 | const Poster = styled.div.attrs({
4 | style: ({ src }) => ({ backgroundImage: `url(${src})` })

```
2. In ItemWrapper.js

```
import styled from "styled-components";
--
2 |  
3 | export default styled.div.attrs({
4 | style: ({ padding, width, height }) => ({ padding, width, height })

```

**Comment by sag1v** [comment]
Thank you, These files are not part of the library but part of the docs, but i would love it if you could create a PR with the fix if you are up for it 🙂

**Comment by sanishkr** [comment]
I will do it as soon as I get some time. Thanks.

**Comment by sanishkr** [comment]
I have sent a PR. Please check

**Comment by sag1v** [comment]
@sanishkr Thanks!

**Comment by sanishkr** [comment]
Hi @sag1v , thanks for merging the PR. Can you please make an NPM release with this? It keeps giving long warning messages in console while running.

**Comment by sag1v** [comment]
@sanishkr No need for an NPM release as its only used in the documentations, I did however re-published the docs. So now you should not see the warnings anymore.
If you still see warnings let me know.
With that said, Now we have a different issue that some examples are broken due to `Docz` new version.

**Comment by sanishkr** [comment]
I still get those warnings, not in the build of this repo, but while using it as an npm package in my project. I don't think that will go away without a new npm release.

**Comment by sag1v** [comment]
@sanishkr You are right I'm sorry! I just noticed you also changed the `ItemWrapper` which is indeed used by the library. I published it under version `0.3.1`. 
Thanks for the help :)

**Comment by sanishkr** [comment]
Happy to help. Thanks for quick publish, @sag1v  👍
